doc/gov/chapters/chapter03.md

```diff
@@ -1,18 +1,41 @@
 [<< Back](../)
 
-# 3. Chapter 3
+# 3. Org Structure and Work Streams
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
 * [3.1 Introduction](#3.1)
-* [3.2 Heading](#3.2)
-* [3.3 New WS Policy](#3.3)
+* [3.2 Org Structure](#3.2)
+* [3.3 Work Streams](#3.3)
+* [3.4 New Work Stream Policy](#3.4)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
 <a name="3.2"></a>
-## 3.2 Heading
+## 3.2 Org Structure
 
 <a name="3.3"></a>
-## 3.3 Heading
+## 3.3 Work Streams
+
+<a name="3.4"></a>
+## 3.4 New Work Streams Policy
+In order for CNTT to stay current with new trends in the industry and the introduction of new technologies and processes, there will be scenarios where a new WS for a new specification work is needed to be created within CNTT. Following is CNTT policy of creating a new WS:
+
+- A new Work Stream **Proposal** has to be created and should contain the following information:
+  - The nature of the Work Stream proposed (Architecture, Implementation, Certification, else).
+  - Detailed scope of the proposed Work Stream.
+  - The rationale behind the need for a new Work Stream instead of making it part of an existing Work Stream.
+  - The relationship to other Work Streams.
+  - The impact on other Work Streams.
+  - The relationship to open-source communities and standard setting organisations (if any).
+  - Proposed Roadmap.
+  - List of committed contributors (at least five) and committed named Work stream Leads and Co-Leads.
+  - Endorsement from at least 2 service providers and 2 vendors.
+- New Work Stream proposals needs to be approved by the Technical Steering Committee during their regular meetings.
+- New Work Stream proposals needs to be approved and signed off by Governance Steering Committee during their regular meetings.
+
+
+
```

Comment:
blank lines should be removed

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/897/commits/e5835b269e040faae081025786a7309b7ec81c08

Reply:
@collivier please resolve